src/Symfony/Component/Mailer/Transport/AbstractTransport.php

```diff
@@ -65,6 +65,8 @@ public function send(RawMessage $message, Envelope $envelope = null): ?SentMessa
         }
 
         if (!$envelope->getRecipients()) {
+            $this->getLogger()->notice('There are no recipients in the mailer envelope. Nothing was sent.');
```

Comment:
I would expect a `LogicException` and a hint what I can do to fix the problem.

Reply:
@jschaedl i did not know the logic behind the « ?SentMessage » method return signature

that is why i added a log and not thrown ex and remove the return null

Maybe we need to wait other review comment :)

Reply:
friendly ping @fabpot as you have commented on related issue https://github.com/symfony/symfony/issues/33753